### ISSUE TITLE
DaprInvocationHttpClient should correctly populate headers in InvokeMethodResponse

### ISSUE THREAD

**Issue by ChrisFajardo-TRI** [dapr/python-sdk#398]
## Expected Behavior

DaprInvocationHttpClient should correctly populate headers in InvokeMethodResponse

## Actual Behavior

DaprInvocationHttpClient does not correctly populate headers in InvokeMethodResponse

## Steps to Reproduce the Problem

Environment:
Self-Hosted
Mac OS 12.3.1
CLI version: 1.7.0
Runtime version: 1.7.0
dapr python sdk: 1.6

Set up a dapr sidecar and HTTP service with a valid endpoint.
Then use the python api to invoke. Headers will not be populated.
```
with DaprClient() as d:
     print(d.invoke_method('service',method','').headers)
```

I suspect the issue is in [this code](https://github.com/dapr/python-sdk/blob/d783ed2538bac6b464a5f1678b202e9edf69d7c8/dapr/clients/http/dapr_invocation_http_client.py#L101)

**Comment by berndverst** [comment]
@ChrisFajardo-TRI I just released Python SDK 1.6.1 which cherry-picked the fix into the existing 1.6 release.

Please note:

The headers field is of type `MetadataDict` which really is a `Dict[List[Union[str, bytes]]`.

What this means is that when you access a header like `Content-Type` you will find that the value is a list. The first value will be the value you are looking for.

So...

```python
res = DaprResponse()
res.headers['Content-Type'][0]  # could be 'application/json' etc
```

**Comment by ChrisFajardo-TRI** [comment]
Thanks @berndverst 
It works!
